### PR TITLE
Misc. Chaplain Additions

### PR DESCRIPTION
**Pull request by SgtManGuy**
## Labels
[CLOTHING] [MAPPING] [REWORK]

## About the PR 
Adds a new chaplain vending machine that replaces the chaplain locker while keeping it's contents.
adds 5 new outfits and 12 new prop books to the new vendor.

## Why's this needed? 
Chaplain is a role that's lacking in a few areas and relies heavily upon the player creating their own fun. Hopefully, these new items will make that east for chaplains to do so.
The new books don't function like the current bible since I feel it'd be unwise to either alter or add new items that function in the way the bible does.
Skeleton book sprites by Rubi. X, red/white, red/dark, blue/white book sprites by Paco. Descriptions by Large Amounts of Screaming and GibusGame, assistance from Tyrant. 

## Changelog 

```changelog
(u)What/SgtManGuy, Tyrant, Large Amounts of Screaming, Rubi, Paco, GibusGame
(*)Replaces the chaplain's locker with a vending machine.
(+)The new vending machine contains all of the old items.
(+)There are 5 new outfits and 12 new prop books in the vending machine.
```

### CODE REVIEW

**Comment by SgtManGuy** [comment]
Huh. Sometime got messed up when when I made the pr. gotta fix that.

**Comment by SgtManGuy** [comment]
[image](https://github.com/goonstation/goonstation/assets/14314992/e430fc71-5946-4dd5-b62b-cb6bfbaab6ef)
image

**Comment by Sord213** [comment]
Screenshot_794

**Comment by SgtManGuy** [comment]
ough. no idea how i missed that

File: code/__secret_public.dme
Comment (Laboredih123) [inline code/__secret_public.dme:1]: yo i dont think this belongs here

File: code/obj/item/scripture.dm [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+	attack_self(mob/user)
+		if (user.find_in_hand(src))
+			if (!src.open)
+				src.open(user)
+			else
+				src.close(user)
+			user.update_inhands()
+		else
+			return ..()
+		return
```
Comment (Laboredih123) [inline code/obj/item/scripture.dm:22]: unneeded return

File: code/obj/item/scripture.dm [hunk 2]
```diff
@@ -0,0 +1,124 @@
...
+
+/obj/item/scripture
+	name = "blank scripture"
+	icon = 'icons/obj/writing.dmi'
+	icon_state = "blankhb"
+	inhand_image_icon = 'icons/mob/inhand/hand_books.dmi'
+	item_state = "blankhb"
+	desc = "An interpretive holy scripture touting… just about whatever you want, really."
+	var/item_state_base = "blankhb"
+	var/open = 0
```
Comment (Laboredih123) [inline code/obj/item/scripture.dm:11]: TRUE and FALSE macros are preferred for boolean vars over 1 and 0.

File: code/obj/item/scripture.dm [hunk 3]
```diff
@@ -0,0 +1,124 @@
...
+			return ..()
+		return
+
+	proc/open (mob/user as mob)
+		src.open = 1
+		icon_state = "[icon_state]open"
+		item_state = "[item_state]open"
+
+	proc/close (mob/user as mob)
+		src.open = 0
```
Comment (Laboredih123) [inline code/obj/item/scripture.dm:30]: Ditto on True and False.

**Review by Laboredih123 — CHANGES_REQUESTED** [review]
Some minor things noticed.

File: code/__secret_public.dme
Comment (ZeWaka) [reply code/__secret_public.dme:1]: deffo not

Comment (SgtManGuy) [reply code/__secret_public.dme:1]: no clue how that got there, whoops. Lemme fix that

**Comment by SgtManGuy** [comment]
Gonna close this and fix some shit in a second pr

**Pull request closed without merging**